src/resolvers/fossil.cr

```diff
@@ -212,10 +212,18 @@ module Shards
       end
     end
 
+    def available_tags : Array(String)
+      tags = capture("fossil tag list -R #{Process.quote(local_fossil_file)}")
+        .split('\n')
+
+      tags.reject(&.empty?)
```

Comment:
**suggestion:** We can directly remove empty entries from `split`.
```suggestion
        .split('\n', remove_empty: true)
```

Ditto below.

Btw. `String#lines` might be more appopriate here, but it doesn't have `remove_empty` (see crystal-lang/crystal#15149). We could still use this with `reject!(&.empty)` though.